src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -1142,13 +1142,14 @@ internal StackFrameVersionOne CreateStackFrameVersionOne(StackFrame v2StackFrame
                 Location location = v2StackFrame.Location;
                 if (location != null)
                 {
-                    string fqln = location.FullyQualifiedLogicalName;
+                    string fqln = location.LogicalLocation?.FullyQualifiedName;
 
                     if (_currentV2Run.LogicalLocations != null &&
-                        !string.IsNullOrWhiteSpace(_currentV2Run.LogicalLocations[location.LogicalLocationIndex].FullyQualifiedName))
+                        location.LogicalLocation != null &&
+                        !string.IsNullOrWhiteSpace(_currentV2Run.LogicalLocations[location.LogicalLocation.ParentIndex].FullyQualifiedName))
```

Comment:
>LogicalLocation.ParentIndex [](start = 91, length = 27)

why did you switch to parent index here? is that right?

Reply:
no, it is not. :) see my other note.

---
In reply to: [264981436](https://github.com/Microsoft/sarif-sdk/pull/1347#discussion_r264981436) [](ancestors = 264981436)